exercises/practice/nth-prime/.docs/instructions.append.md

```diff
@@ -1 +1,3 @@
+# Instructions append
```

Comment:
There are several instances like this; I suspect we don't _actually_ want titles like "Instructions append". As I understand it, those'll be stripped anyway though, so I don't want to block this PR on it.

Reply:
They will indeed be stripped.